libafl/src/events/mod.rs

```diff
@@ -377,13 +378,15 @@ where
                 },
             )?;
 
-            if let Some(x) = state.stability() {
-                let stability = f64::from(*x);
+            if state.has_metadata::<UnstableEntriesMetadata>() {
```

Comment:
This is an additional (quite costly) HashMap lookup and shouldn't be used if you will immediately get the value anyway